File-Formats/PDF/Convert-HTML-To-PDF/azure.md

```diff
@@ -174,139 +162,134 @@ A complete work sample for converting an HTML to PDF in Azure App service on Lin
 
-Add an Export to the PDF button in the index.cshtml.
+5. Add an Export to the PDF button in the index.cshtml.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-<div class="btn">
+   <div class="btn">
     @{ Html.BeginForm("ExportToPDF", "Home", FormMethod.Post);
         {
             <input type="submit" value="Export To PDF" class=" btn" />
         }
-}
-</div>
+    }
+   </div>
 
-{% endhighlight %}
+   {% endhighlight %}
 
 ![Convert HTMLToPDF Azure Docker Step5](htmlconversion_images/DockerStep5.png)
 
-Include the following namespaces and code samples in the controller for converting HTML to PDF.
+6. Include the following namespaces and code samples in the controller for converting HTML to PDF.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-using Syncfusion.HtmlConverter;
-using Syncfusion.Pdf;
-using System.IO
+   using Syncfusion.HtmlConverter;
+   using Syncfusion.Pdf;
+   using System.IO
 
-{% endhighlight %}
+   {% endhighlight %}
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-public ActionResult ExportToPDF()
+   public ActionResult ExportToPDF()
 {
      //Initialize HTML to PDF converter. 
      HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
- 
      BlinkConverterSettings settings = new BlinkConverterSettings();
-     
      //Set command line arguments to run without the sandbox.
      settings.CommandLineArguments.Add("--no-sandbox");
      settings.CommandLineArguments.Add("--disable-setuid-sandbox");
-     
      //Assign Blink settings to the HTML converter.
      htmlConverter.ConverterSettings = settings;
- 
      //Convert URL to PDF.
      PdfDocument document = htmlConverter.Convert("https://www.syncfusion.com");
- 
      MemoryStream stream = new MemoryStream();
- 
      //Save and close a PDF document. 
      document.Save(stream);
- 
      return File(stream.ToArray(), System.Net.Mime.MediaTypeNames.Application.Pdf, "URL_to_PDF.pdf");
 }
 
-{% endhighlight %}
+   {% endhighlight %}
 
-Build and run the sample in the docker, it will pull the Linux docker image from the docker hub and run the project. Now, the webpage will open in the browser. Click Export to PDF option to convert the Syncfusion webpage to a PDF.
+7. Build and run the sample in the docker, it will pull the Linux docker image from the docker hub and run the project. Now, the webpage will open in the browser. Click Export to PDF option to convert the Syncfusion webpage to a PDF.
 ![Convert HTMLToPDF Azure Docker Step6](htmlconversion_images/AzureDocker2.png)
 
-By executing the program, you will get the PDF document as follows.
+8. By executing the program, you will get the PDF document as follows.
 ![Convert HTMLToPDF Azure Docker Step7](htmlconversion_images/htmltopdfoutput.png)
 
-### Deploy the container to Azure container instance:
+## Deploy the container to Azure container instance:
```

Comment:
This is third level heading only..
### Deploy the container to Azure container instance - Pls use this